core/model/src/main/java/com/yapp/model/Sessions.kt

```
@@ -0,0 +1,16 @@
+    val id: String,
+    val name: String,
+    val place: String?,
+    val date: String,
+    val endDate: String?,
+    val time: String?,
+    val type: AttendType,
+    val progressPhase: String
+) {
+    enum class AttendType {
+
+    }
```

detekt (1.23.7)

[warning] The class or object AttendType is empty.

(detekt.empty-blocks.EmptyClassBlock)